azerothcore/azerothcore-wotlk
# fix(DB/Creature): Maker's Overlook

## CHANGES PROPOSED:

Fix complete zone Maker's Overlook as barely all NPC's were missing

- added missing creatures
- added missing waypoint data
- fixed InhabitTypes 

**Source:** 
- TrinityCore DB (Branch 3.3.5)

**Research:** 
- YoutTube Video from Live WotLK https://www.youtube.com/watch?v=iCmwn9vzMlo&ab_channel=wtbblue


### Titan Sentry - Wrong InhabitatType before PR

![image](https://user-images.githubusercontent.com/55922592/98987728-ed96cc80-2526-11eb-8c10-2dd2719d5c51.png)

### Titan Sentry - With  PR

![image](https://user-images.githubusercontent.com/55922592/98987801-056e5080-2527-11eb-9e14-ef34b2319f5d.png)

### Sholazar Guardian - Wrong position and missing waypoint data before PR

![image](https://user-images.githubusercontent.com/55922592/98987907-28990000-2527-11eb-856e-9a86242b5270.png)

### Sholazar Guardian - With PR

![image](https://user-images.githubusercontent.com/55922592/98988022-4ebea000-2527-11eb-9ca7-b5682b62e699.png)

### Titan Sentry - Wrong InhabitatType before PR

![image](https://user-images.githubusercontent.com/55922592/98988194-92b1a500-2527-11eb-9478-2c335727fd26.png)

### Titan Sentry - With  PR

![image](https://user-images.githubusercontent.com/55922592/98988238-a3621b00-2527-11eb-933b-8b40cfb8b9ad.png)

### Overlook Sentry - Missing creature and waypoint data before PR

![image](https://user-images.githubusercontent.com/55922592/98988313-c096e980-2527-11eb-9c66-528733a931d5.png)

### Overlook Sentry - With PR

![image](https://user-images.githubusercontent.com/55922592/98988360-d3a9b980-2527-11eb-982a-15e6e55aa1f1.png)

### Sholazar Guardian - Missing creatures and waypoint data before PR

![image](https://user-images.githubusercontent.com/55922592/98988482-0784df00-2528-11eb-8ee2-28cbeeb6e1b2.png)

### Sholazar Guardian - With PR

![image](https://user-images.githubusercontent.com/55922592/98988622-3307c980-2528-11eb-824e-8699e4e20338.png)

### Sholazar Guardian - Missing creatures before PR

![image](https://user-images.githubusercontent.com/55922592/98988676-46b33000-2528-11eb-9cb0-1bd8a0d7acdd.png)

### Sholazar Guardian - With PR

![image](https://user-images.githubusercontent.com/55922592/98988720-5763a600-2528-11eb-8f37-7ad57eb00e65.png)

## ISSUES ADDRESSED:
- None



## TESTS PERFORMED:
- Checked existing waypoint data - none exists
- Imported SQL
- Checked every creature ingame (See the Screenshots)




## HOW TO TEST THE CHANGES:


1. .go c 111422
2. Check the area bevore you import the SQL
3. Import SQL
4. Check the creatures in the area


## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
